t/server/controller/bad_request.t

```diff
@@ -0,0 +1,55 @@
+use strict;
+use warnings;
+use lib 't/lib';
+
+use YAML::XS;
+use Cpanel::JSON::XS ();
+use MetaCPAN::Server ();
+use MetaCPAN::TestHelpers;
+use Plack::Test;
+use Test::More;
+use Ref::Util qw( is_arrayref is_hashref );
+
+
+my $app  = MetaCPAN::Server->new->to_app();
+my $test = Plack::Test->create($app);
+
+my $bigquery = YAML::XS::LoadFile($FindBin::Bin . '/../../../test-data/big-result.yml');
+my $sbigqueryjson = Cpanel::JSON::XS::encode_json($bigquery);
```

Comment:
I think this logic could be simplified. Rather than using fixture data, we could just have `JSON` in a heredoc. That would eliminate parsing the `YAML` as well as having to load a file.

Reply:
I created an inline `JSON` string and removed the unused dependencies.